### PR TITLE
[Pipelined]Add GHZ gRPC testing benchmark

### LINKED ISSUE #5012: [Reliability] Benchmarking performance between stateful and stateless modes
We need a concrete way to compare the performance on releases / versions of the platform using stateful and stateless modes, we can use scale testing on Spirent setups to get results of:

- Attach rate of UEs per second
- Total UEs attached successfully
- Total UEs detached successfully
- Time of attach process per UE

### PR DESCRIPTION
**Pull request by koolzz**
Signed-off-by: Nick Yurchenko <koolzz@fb.com>

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

**Review by ardzoht — APPROVED** [review]
lgtm

File: lte/gateway/python/load_tests/loadtest_pipelined.py [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+
+PROTO_DIR = 'lte/protos'
+IMPORT_PATH = '/home/vagrant/magma'
+RESULTS_PATH = '/var/tmp'
+
+UEInfo = namedtuple('UEInfo', ['imsi_str', 'ipv4_src', 'ipv4_dst',
+                               'rule_id'])
+
+def _gen_ue_set(num_of_ues):
+    imsi = 123000000
```
Comment (pshelar) [inline lte/gateway/python/load_tests/loadtest_pipelined.py]: can we have single definition between load-test and pipelined_cli ?

Comment (koolzz) [reply lte/gateway/python/load_tests/loadtest_pipelined.py]: I can probably just import it, will fix

Comment (koolzz) [reply lte/gateway/python/load_tests/loadtest_pipelined.py:49]: sure let me import it

**Pull request merged**